### ISSUE TITLE
EasyCLA migration

### ISSUE THREAD

**Issue by mrbobbytables** [kubernetes/org#2778]
The CNCF is ready to cut over to EasyCLA

Right now the target timeframe is just after code-freeze (7/10) when activity in k/k should be minimal.
The validation plan will include verifying org members exist in both systems, in addition to a random selection of contributors, along with running the check against a list of PRs.

/assign @mrbobbytables @nikhita @dims 
For main points of contact.

Planning Doc: https://docs.google.com/document/d/1DYSysEYUvLFq_Jhx7QM5AAKhFEKN9cjm27vt1ZKk_Nc/edit

---
Hi Nikhita, Bob, Amye, and Chris,

I'm working with Pranab and team to support your migration onto EasyCLA v2, and I've created a rough plan for what this could look like - Please take a look and let me know what you think.
 LFX EasyCLA v2 - Kubernetes | Go-to-Market Plan
This is pretty similar to what we did for the pilot and general releases, which worked very well. Ultimately, our goal is to make sure this is as smooth a transition as possible for your community, so I hope you find this plan helpful.

I'd like to schedule some time for us to review this early next week. Would any of the following times work for you (all times in PT and IST)?
Mon 5/24 - Between 10am - 1pm PT (10:30pm - 1:30 am IST)
Tue 5/25 - After 3pm PT (Morning next day, IST)
Wed 5/26 - 11:30am PT or after 3pm PT (12am or after 3:30am next day, IST)
**@Nikhita - I understand you're based in India, so please let me know what hours are preferred for you and I'll do my best to accommodate.

Thanks!
Stephanie

**Comment by mrbobbytables** [comment]
Migration has been pushed back to the 2nd week of August, at this time here are the current known tasks excluding messaging that must be completed before a potential roll-out.
- [ ] Verify Kubernetes Org Members exist in both systems
  - [ ] LF given a dump of Org Members
  - [ ] Verified against their cla DB
- [ ] Confirm Branch Protection configuration with LF
- [ ] EasyCLA support by the prow plugin (https://github.com/kubernetes/test-infra/issues/22721)
- [ ] Configure and test with contributor-playground repo

**Comment by mrbobbytables** [comment]
During the org audit multiple members did not come up with an association in the LF CLA DB. Below are the active contributors from the audit that having a missing CLA association.

If you are on this list, please reply here, or reach out to me directly in slack / email killen.bob[@]gmail.com with your email(s) that might be associated with your cla, and if at all possible your lfid. With that we should be able to get it squared away before migration.

---
- [ ] @gabbifish
- [ ] @gracenng
- [ ] @haibinxie
- [ ] @hkamel
- [ ] @kendallnelson
- [ ] @kgroschoff
- [ ] @kinarashah
- [ ] @priyankah21
- [ ] @pushkarj
- [ ] @reylejano
- [ ] @xichengliudui
- [ ] @dgrisonnet
- [ ] @helight
- [ ] @howieyuen
- [ ] @moshitrit
- [ ] @prezha
- [ ] @wangyx1992
- [ ] @yuanchen8911

**Comment by dims** [comment]
@mrbobbytables i've pinged everyone on slack as well. 🤞🏾

**Comment by mrbobbytables** [comment]
Thanks @dims <3

**Comment by howieyuen** [comment]
> During the org audit multiple members did not come up with an association in the LF CLA DB. Below are the active contributors from the audit that having a missing CLA association.
> 
> If you are on this list, please reply here, or reach out to me directly in slack / email killen.bob[@]gmail.com with your email(s) that might be associated with your cla, and if at all possible your lfid. With that we should be able to get it squared away before migration.
> 
> * [x]  @gabbifish
> * [x]  @gracenng
> * [ ]  @haibinxie
> * [x]  @hkamel
> * [x]  @kendallnelson
> * [x]  @kgroschoff
> * [ ]  @kinarashah
> * [ ]  @PriyankaH21
> * [x]  @PushkarJ
> * [x]  @reylejano
> * [ ]  @xichengliudui
> * [x]  @dgrisonnet
> * [ ]  @helight
> * [ ]  @howieyuen
> * [ ]  @MoShitrit
> * [x]  @prezha
> * [ ]  @wangyx1992
> * [ ]  @yuanchen8911

hello, what should I do ?

**Comment by wangyx1992** [comment]
The same question as @howieyuen.

**Comment by dims** [comment]
@wangyx1992 @howieyuen there are instructions from @mrbobbytables :) he needs the following info.

>  please reply here, or reach out to me directly in slack / email killen.bob[@]gmail.com with your email(s) that might be associated with your cla, and if at all possible your lfid.

**Comment by mrbobbytables** [comment]
@dims has it right, I just need your email address that you used to sign the cla and if at all possible your lfid. The WORST case is post migration you will have to sign the CLA again.

**Comment by helight** [comment]
I am helight, my CLA mail ***@***.***




------------------&nbsp;原始邮件&nbsp;------------------
发件人:                                                                                                                        "kubernetes/org"                                                                                    ***@***.***&gt;;
发送时间:&nbsp;2021年7月3日(星期六) 凌晨0:37
***@***.***&gt;;
***@***.******@***.***&gt;;
主题:&nbsp;Re: [kubernetes/org] EasyCLA migration (#2778)





 
During the org audit multiple members did not come up with an association in the LF CLA DB. Below are the active contributors from the audit that having a missing CLA association.
 
If you are on this list, please reply here, or reach out to me directly in slack / email killen.bob[@]gmail.com with your email(s) that might be associated with your cla, and if at all possible your lfid. With that we should be able to get it squared away before migration.
 
  
 @gabbifish
 
 @gracenng
 
 @haibinxie
 
 @hkamel
 
 @kendallnelson
 
 @kgroschoff
 
 @kinarashah
 
 @PriyankaH21
 
 @PushkarJ
 
 @reylejano
 
 @xichengliudui
 
 @dgrisonnet
 
 @helight
 
 @howieyuen
 
 @MoShitrit
 
 @prezha
 
 @wangyx1992
 
 @yuanchen8911

**Comment by howieyuen** [comment]
I am howieyuen, my CLA email is haoyuan@mail.ustc.edu.cn

**Comment by wangyx1992** [comment]
I am wangyx1992，my CLA email is wang.yixiang@zte.com.cn

**Comment by helight** [comment]
I am helight，my CLA email is helightxu@tencent.com

**Comment by mrbobbytables** [comment]
ref: https://github.com/kubernetes/test-infra/issues/22721#issuecomment-926032214
Here is the current plan:

1) Enable EasyCLA as non-blocking org wide next week (accompanied by comms)
2) Assuming no issues are reported, we cut-over after code freeze (Nov 16th). This means updating the current CLA plugin to look for `EasyCLA` instead of `cla/linuxfoundation`, removing our help text, and potentially updating it to also look for `/easycla` in the comments. The LF will also perform their final sync of the DBs between systems.
3) Given the time, most PRs should be updated with an `EasyCLA` status. We script hitting open PRs to refresh CLA status on the remaining PRs.

If there is an issue, we revert to the old system. Anyone that signed up after cut over may have to sign the old CLA.

**Comment by brendandburns** [comment]
@mrbobbytables I'm seeing discrepancies between the old CLA bot and EasyCLA, e.g.

https://github.com/kubernetes-client/c/pull/93

**Comment by mrbobbytables** [comment]
@brendandburns thats weird...it shouldn't be blocking, we have prow setup to ignore to the EasyCLA context

**Comment by mrbobbytables** [comment]
Found the issue, it was only setup to be ignored in kubernetes 🤦

**Comment by sftim** [comment]
(low priority)
How's this migration going?

**Comment by mrbobbytables** [comment]
Current plan is to migrate the weekend of Feb 5th, right after Enhancements freeze. comms will be going out later today
